setup.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+raise DeprecationWarning('Use rez-build to build this package at Rodeo.')
```

Comment:
Why ?

we still want to keep the possibility to use normal python distribution, pip install possibility, for any of our python code. 

We use for now temporarily package.py + rezbuild.py for our python code simply because rez-pip isn't complete / has missing or incorrect stuffs.. otherwise we would probably use it at 100%.

also, reminder : the way we use package.py for python related things(pure libs or scripts, (at the end it's the same)), *by/with* modifying/adding to PYTHONPATH is no good. we keep that way for now so to have something usable.
but at some point (and very sooner than later) we will change that.. (towards using a sitecustomize.py).